db/1.13/modules/ROOT/pages/database-connector-xa-transactions.adoc

```diff
@@ -0,0 +1,44 @@
+= Using XA Transactions in Database Connector
+
+Anypoint Connector for Database (Database Connector) provides the option to use Extended Architecture Transactions (XA Transactions) when you enable the *Use XA Transactions* field in the Database Connector global configuration.
+
+== Before You Begin
+
+Before getting started, ensure that you have understanding of xref:mule-runtime::xa-transactions.adoc[XA Transactions]
```

Comment:
Before getting started, ensure that you understand xref:mule-runtime::xa-transactions.adoc[XA Transactions].

or

Before getting started, ensure that you have an understanding of xref:mule-runtime::xa-transactions.adoc[XA Transactions].